include/mbedtls/aria.h

```diff
@@ -185,7 +191,7 @@ int mbedtls_aria_crypt_ecb( mbedtls_aria_context *ctx,
  * \param input    The buffer holding the input data.
  * \param output   The buffer holding the output data.
  *
- * \return         \c 0 on success, or #MBEDTLS_ERR_ARIA_INVALID_INPUT_LENGTH
```

Comment:
Should `MBEDTLS_ERR_ARIA_INVALID_INPUT_LENGTH` remain documented as the error returned since we aren't deprecating it anymore?

Reply:
It looks there there may be similar issues with Blowfish and Camelia as well.

Reply:
That's right. I reverted the corresponding change. The documentation is generally in bad shape here, but I deliberately didn't touch that because the parameter validation PRs will take care of these improvements.